src/org/zaproxy/zap/extension/pscanrules/resources/help/contents/pscanrules.html

```diff
@@ -11,24 +11,40 @@
 The following release quality passive scan rules are included in this add-on:
 
 <H2>Application Errors</H2>        
+ Check server responses for HTTP 500 - Internal Server Error type responses or those that contain a known error string.
  
 <H2>Cache Control</H2>
+Checks "Cache-Control" and "Pragma" response headers against general industry best practice settings for protection of sensitive content. 
 
 <H2>Content Type Missing</H2>
+Raises an alert if the response is lacking a Content-Type header or if the header exists but the value is empty.
 
-<H2>Cookie HTTP Only</H2>
+<H2>Cookie HTTPOnly</H2>
```

Comment:
Was this change on purpose? If so, "HttpOnly" instead? Or, the actual scanner name "Cookie No HttpOnly Flag"?